src/lib/Microsoft.Health.Fhir.Ingest.Schema/Measurement.cs

```diff
@@ -29,6 +29,8 @@ public Measurement()
 
         public string CorrelationId { get; set; }
 
+        public string PartitionId { get; set; }

```

Comment:
>PartitionId [](http://example.com/codeflow?start=22&length=11)

I wonder if this is the best way to capture a partition.  We will have a reader per partition on FHIR Conversion.  The partition should be implicit with the import service.  We are essentially replicating the information for each event and I don't think we need to.  

The data freshness calcuation is already using the consuming servicings partition.  It appears this is only being used in the MeasurementFhirImportService so we could add that there as well and remove the need to capture the information here.

One ask is please make sure these changes (and those in the MeasurementFhirImportService) work with the original IoMT Connector using Stream Analytics.  I *believe* the partition id should be part of the payload sent over but if it isn't it will be missing and that might cause errors.

Reply:
Actually this is not being used at all, I forgot to clean this up. I'm directly looking at the event batch and getting the partitionId from there in MeasurementFhirImportService.
As for the "IoMT Connector using Stream Analytics" scenario - I wasn't sure too if there would be partitionId, so I ensured that if there is no partitionId then I am not adding this dimension at all.